style.scss

```diff
@@ -0,0 +1,78 @@
+$darkGreen: #21594e;
+$green: #3db14c;
+$limeGreen: #d9ed14;
+$yellow: #fef97a;
+$red: #a31423;
+$white: #ffffff;
+
+* {
+  font-family: 'Sigmar One', cursive;
+}
+
+h2 {
+  font-size: 22px;
+}
+
+.navbar,
+body,
+.btn:hover,
+.btn:focus {
+  background-color: $darkGreen;
+}
+
+body {
+  background-color: $darkGreen;
+  min-height: 100vh;
+}
+
+.card {
+  background-color: $limeGreen;
+  color: $darkGreen;
+}
+
+.displayCase {
+  font-size: 35px;
+  background-color: $limeGreen;
+  color: $darkGreen;
+  p {
+    text-shadow: 2px 2px $darkGreen, 2px -2px $darkGreen, -2px 2px $darkGreen, -2px -2px $darkGreen;
+    color: $yellow;
+    }
+  .stat {
+    text-shadow: 2px 2px $darkGreen, 2px -2px $darkGreen, -2px 2px $darkGreen, -2px -2px $darkGreen;
+    color: $white;
+  }
+}
+
+.btn {
+  background-color: $red;
+  color: #ffffff;
+}
+
+.btn:hover{
+  color: #ffffff;
+}
+.btn:focus{
+  color: #ffffff;
+}
+
+.card-body {
+  height: 80px;
+  justify-content: space-between;
+}
+
+.row {
+  margin-right: 0;
+}
+
+.dino-img-container {
+  position: relative;
+  width: 250px;
+  height: 250px;
+  overflow: hidden;
+  border: 1px solid $darkGreen;
+  background-repeat: no-repeat;
+  background-position: center top;
+  background-size: cover;
+}
+
```

Comment:
Get a media query on the display case image/ use bootstrap for responsiveness :)